src/wp-includes/class-wp-theme-json-resolver.php

```diff
@@ -151,14 +151,17 @@ public static function get_core_data() {
 	 *
 	 * @since 5.8.0
 	 * @since 5.9.0 Theme supports have been inlined and the `$theme_support_data` argument removed.
+	 * @since 6.0.0 Adds a second parameter to allow the theme data to be returned without theme supports.
 	 *
 	 * @param array $deprecated Deprecated. Not used.
+	 * @param array $settings Contains a key called with_supports to determine whether to include theme supports in the data.
 	 * @return WP_Theme_JSON Entity that holds theme data.
 	 */
-	public static function get_theme_data( $deprecated = array() ) {
+	public static function get_theme_data( $deprecated = array(), $settings = array( 'with_supports' => true ) ) {
```

Comment:
We don't typically follow this pattern in Core. Traditionally this would be an empty array and use `wp_parse_args` to set defaults.

Reply:
Done in 2529def636